lux/executor/PandasExecutor.py

```diff
@@ -435,7 +439,14 @@ def compute_data_type(self, ldf: LuxDataFrame):
                 ldf._data_type[attr] = ldf._type_override[attr]
             else:
                 temporal_var_list = ["month", "year", "day", "date", "time", "weekday"]
-                if is_datetime(ldf[attr]):
+
+                if is_timedelta64_series(ldf[attr]):
+                    ldf._data_type[attr] = "quantitative"
```

Comment:
@dorisjlee probably here.

Reply:
converting the data type metadata here is okay, is there a reason why we don't do it in the compiler?

Reply:
I didn't see a place where to do it, if you can point me to it maybe that what we are looking for.

Reply:
This function is called inside the `Compiler`, so we're all good!